ui/rollSection/rollSection.pug

```diff
@@ -34,11 +34,11 @@
       input#attr_action_description.flexing(name='attr_action_description' type='text')
       label(for='attr_action_feats') Feats
       input#attr_action_feats.narrowish(name='attr_action_feats' type='number' value='0')
-      label(for='attr_action_feats_EP') EP &ensp;
+      label(for='attr_action_feats_EP') PWR &ensp;
       input#attr_action_feats_EP.narrow(name='attr_action_feats_EP' value='@{action_feats}' disabled='true')
-      label(for='attr_action_EP') EP
+      label(for='attr_action_EP') PWR
```

Comment:
The inner variables will be a bit of a pain (`attr_action_EP`, etc.)
I don't know what we should do with that.
Maybe we should follow up with a rename and migration to clone the values to a matching name...

Reply:
That sounds right.

I think it requires three things:
- Code at character sheet open that copies the values under the old names to the new ones.
- Triggers on changes to the old variables that copy to the new ones
- Triggers on changes to the new variables that copy to the old ones.

The last two are so all the character tokens that refer to the old variables will keep working.